Example/package.json

```diff
@@ -22,7 +22,7 @@
     "pod-install": "cd ios && pod install && cd .."
   },
   "dependencies": {
-    "@babel/plugin-proposal-export-namespace-from": "^7.17.12",
+    "@babel/plugin-proposal-export-namespace-from": "^7.18.9",
```

Comment:
The `@babel/plugin-proposal-export-namespace-from` plugin is required only for the Web platform. This plugin is included in `@babel/preset-env` - [source](https://babeljs.io/docs/en/babel-plugin-proposal-export-namespace-from). However, the preset `@babel/preset-env` is often added by default for Web apps.

> Do we need this to be a direct dependency in the Example app?

We need the plugin for the Web version of the Example app.

> Do all RN apps that use Reanimated need to install this plugin explicitly?

No, just web apps if they don't have `@babel/preset-env` preset.

> Why can't we make a dependency of react-native-reanimated?

We can, but I think this is pointless because even if we add the plugin to the dependency, users still need to add the plugin to the babel config file.